crates/python/qcs_sdk/qvm/api.pyi

```diff
@@ -36,17 +36,6 @@ class AddressRequest:
     """
 
     def __new__(cls, input: Union[bool, Sequence[int]]): ...
-    def inner(self) -> Union[bool, Sequence[int]]: ...
-    @staticmethod
-    def from_all(all: bool) -> AddressRequest: ...
-    @staticmethod
-    def from_indices(indices: Sequence[int]) -> AddressRequest: ...
-    def as_all(self) -> Optional[bool]: ...
-    def as_indices(self) -> Optional[Sequence[int]]: ...
-    def to_all(self) -> bool: ...
-    def to_indices(self) -> Sequence[int]: ...
-    def is_all(self) -> bool: ...
-    def is_indices(self) -> bool: ...
```

Comment:
I would personally prefer having static methods `include_all`, `exclude_all`, and `from_indices` as constructors, rather than a `__new__` that takes a boolean or a list of integers. I'm fine with the rest of the methods not existing, though, if we don't expect users to inspect the `AddressRequest` after creating it.

Reply:
Okay, changed them to static methods.